input/pagecontent/comparison.md

```diff
@@ -1,1206 +1,506 @@
+            <td style="width: 25%;">IPS requires minimum of 1.</td>
+        </tr>
+        <tr>
+            <td style="width: 25%;">Observation.performer</td>
+            <td style="width: 25%;">IPS requires minimum of 1.</td>
+        </tr>
+        <tr>
+            <td style="width: 25%;">Observation.value</td>
+            <td style="width: 25%;">IPS requires UCUM for coded quantity units.</td>
+        </tr>
+        <tr>
+            <td rowspan="4" style="width: 25%;"><a href="https://hl7.org/fhir/uv/ips/STU1.1/StructureDefinition-Observation-results-pathology-uv-ips.html">Observation Results: pathology (IPS)</a></td>
+            <td style="width: 25%;">Observation.status</td>
+            <td style="width: 25%;">IPS requires value 'final'.</td>
+        </tr>
+        <tr>
+            <td style="width: 25%;">Observation.subject.reference</td>
+            <td style="width: 25%;">IPS requires minimum of 1.</td>
+        </tr>
+        <tr>
+            <td style="width: 25%;">Observation.performer</td>
+            <td style="width: 25%;">IPS requires minimum of 1.</td>
+        </tr>
+        <tr>
+            <td style="width: 25%;">Observation.value</td>
+            <td style="width: 25%;">IPS requires UCUM for coded quantity units.</td>
+        </tr>
+        <tr>
+            <td rowspan="2" style="width: 25%;"><a href="StructureDefinition-au-core-procedure.html">AU Core Procedure</a></td>
+            <td rowspan="2" style="width: 25%;"><a href="https://hl7.org/fhir/uv/ips/STU1.1/StructureDefinition-Procedure-uv-ips.html">Procedure (IPS)</a></td>
+            <td style="width: 25%;">Procedure.subject.reference</td>
+            <td style="width: 25%;">IPS requires minimum of 1.</td>
+        </tr>
+        <tr>
+            <td style="width: 25%;">Procedure.performed[x]</td>
+            <td style="width: 25%;">IPS requires minimum of 1.</td>
+        </tr>
+        <tr>
+            <td rowspan="3" style="width: 25%;"><a href="StructureDefinition-au-core-smokingstatus.html">AU Core Smoking Status</a></td>
+            <td rowspan="3" style="width: 25%;"><a href="https://hl7.org/fhir/uv/ips/STU1.1/StructureDefinition-Observation-tobaccouse-uv-ips.html">Observation - SH: tobacco use</a></td>
+            <td style="width: 25%;">Observation.subject.reference</td>
+            <td style="width: 25%;">IPS requires minimum of 1.</td>
+        </tr>
+        <tr>
+            <td style="width: 25%;">Observation.code</td>
+            <td style="width: 25%;">IPS requires LOINC code 72166-2.</td>
+        </tr>
+        <tr>
+            <td style="width: 25%;">Observation.value[x]</td>
+            <td style="width: 25%;">IPS requires value from <a href="https://hl7.org/fhir/uv/ips/STU1.1/ValueSet-current-smoking-status-uv-ips.html">Current Smoking Status - IPS</a>.</td>
+        </tr>
+    </tbody>
+</table>
+
+##### Missing Data
+An AU Core compliant resource meets the requirements for Missing Data in the compared IPS profile(s).
```

Comment:
To my knowledge IPS does not contain requirements on handling Missing Data. If there are some please send me a link to them.

Otherwise for the IPS Missing Data & Supressed Data items:
1. Replace "An AU Core compliant resource meets the requirements for Missing Data in the compared IPS profile(s)." with "IPS does not have Suppressed Data requirements."
2. Replace "does not contain any requirements on Suppressed Data" with "does not include requirements for Suppressed Data".

Reply:
https://hl7.org/fhir/uv/ips/STU1.1/design.html#missing-data. I only found it the other day.

Reply:
1. Replaced with "AU Core compliant resources are compliant with IPS requirements for Missing Data." - made consistent with the IPA & AU Core Missing Data statements
2. Done